code/__HELPERS/roundend.dm

```diff
@@ -359,20 +359,48 @@
 /client/proc/roundend_report_file()
 	return "data/roundend_reports/[ckey].html"
 
-/datum/controller/subsystem/ticker/proc/show_roundend_report(client/C, previous = FALSE)
+/**
+ * Log the round-end report as an HTML file
+ *
+ * Composits the roundend report, and saves it in two locations.
+ * The report is first saved along with the round's logs
+ * Then, the report is copied to a fixed directory specifically for 
+ * housing the server's last roundend report. In this location, 
+ * the file will be overwritten at the end of each shift.
+ */
+/datum/controller/subsystem/ticker/proc/log_roundend_report()
+	var/filename = "[GLOB.log_directory]/round_end_data.html"
+	var/list/parts = list()
+	parts += "<div class='panel stationborder'>"
+	parts += GLOB.survivor_report
+	parts += "</div>"
+	parts += GLOB.common_report
+	var/content = parts.Join()
+	//Log the rendered HTML in the round log directory
+	fdel(filename)
+	text2file(content, filename)
+	//Place a copy in the root folder, to be overwritten each round. 
+	filename = "data/last_roundend/server_last_roundend_report.html"
+	fdel(filename)
+	text2file(content, filename)
+
+/datum/controller/subsystem/ticker/proc/show_roundend_report(client/C, your_last_round = FALSE, server_last_round = FALSE)
 	var/datum/browser/roundend_report = new(C, "roundend")
 	roundend_report.width = 800
 	roundend_report.height = 600
 	var/content
 	var/filename = C.roundend_report_file()
-	if(!previous)
+	if(your_last_round) //Look at this player's last round
+		content = file2text(filename)
+	else if (server_last_round) //Look at the last round that this server has seen
+		content = file2text("data/last_roundend/server_last_roundend_report.html")
```

Comment:
Let me know if I should have this string stored as a variable somewhere.

Reply:
What's the point of the directory here.

Reply:
In my head I thought this would be cleaner for some reason 🥴